internal/pkg/template/templates/workloads/partials/cf/https-listener.yml

```diff
@@ -38,8 +38,13 @@ LoadBalancerDNSAlias{{$hostedZoneID}}:
           HostedZoneId: !GetAtt EnvControllerAction.InternalLoadBalancerHostedZone
           DNSName: !GetAtt EnvControllerAction.InternalLoadBalancerDNSName
           {{- else}}
+            {{- if $.CDNEnabled}}
```

Comment:
Should we or is it possible for us to create another resource for this record, instead of fitting it into this resource that was designed for Load Balancer DNS?

Reply:
the thing is they could use the same alias. For example: `example.com` is used for the service and the target is LB DNS name. We can't create another resource with the same `example.com` pointing to the CF without deleting this resource first.